src/App.js

```diff
@@ -1,12 +1,104 @@
 import React from "react";
-import Form from "./Form";
 import "./App.css";
+import {useState} from "react"
+import Testing from "./testing"
 
 function App() {
+
+ const [number, setNumber] = useState("")
+ const [typeOf, setTypeOf] = useState("")
+ const [result,setResult]  = useState("")
+  
+  const[name,setName] = useState("Florence")
+ 
+ 
+ function handleSubmit(event) {
+    event.preventDefault()
+   
+   if (typeOf === "sum") {
+     setResult( number.split(",").map((num)=> Number(num)).reduce((a,b)=> a+b,0)   )
+   }
+   if (typeOf === "average") {
+    setResult(number.split(",").map((num)=> Number(num)).reduce((a,b)=> a+b,0) / number.split(",").length)
+   }
+   if (typeOf === "mode") {
+    
+      let ourarray = number.split(",").map((num)=> Number(num))
+      let obj = {}
+
+      ourarray.forEach(numb => {
+        if (!obj[numb]) {
+          obj[numb] = 1
+        } else {
+          obj[numb] += 1
+        }
+      })
+ 
+       let highestValue = 0;
+       let highestValueKey = -10000;
+       for (let key in obj) {
+        const value = obj[key]
+        if ((value >= highestValue) && (Number(key) > highestValueKey)) {
+           highestValue = value
+           highestValueKey = Number(key)
+        }
+       }
+
+
+    setResult(highestValueKey)
+
+
+
+   }
+   if (typeOf === "") {
+    setResult("Invalid input.")
+   }
+
+   
+
+    
+
+  
+  }
+  
+  function handleSelectChange(event) {
+   
+   
+    setNumber(event.target.value)
+   
+  }
+
+  function handleChangeSelect( event) {
+    
+    setTypeOf(event.target.value)
+
+
+  }
+   
```

Comment:
it looks like you don't have any code resetting your form after the user submits